src/store/modules/order/mutations.ts

```diff
@@ -8,9 +8,14 @@ const mutations: MutationTree <OrderState> = {
     state.purchaseOrders.total = payload.total;
   },
   [types.ORDER_CURRENT_UPDATED](state, payload) {
-    state.current.orderId = payload.order[0]?.orderId;
-    state.current.externalOrderId = payload.order[0]?.externalOrderId;
-    state.current.items = payload.order
+    if(payload.order){
```

Comment:
Why this is changed?

Reply:
Initially when we are setting the current order we are passing an object that contains an array order as payload but when we are updating the quantity accepted of an item we are passing state.current as payload.